node-notes-cli/app.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const data = require('./data.json');
+
+if (process.argv[2] === 'read') {
+  for (const key in data.notes) {
+    console.log(`${key}: ${data.notes[key]}`);
+  }
+} else if (process.argv[2] === 'create') {
+  let last = 0;
+  for (const key in data.notes) {
+    if (parseInt(key) > last) {
+      last = parseInt(key);
+    }
+  }
+  data.notes[last + 1] = process.argv[3];
```

Comment:
This can be simplified by incrementing `data.nextId` and using that as the note id/key.